generic/ltp.py.data/README.txt

```diff
@@ -0,0 +1,9 @@
+Single Host Configuration:
+It's the default configuration (if the RHOST environment variable is not defined). LTP adds ltp_ns network namespace and auto-configure veth pair according to LTP network environment variables.
+
+Two Host Configuration:
+This setup requires the RHOST environment variable to be set properly and configured SSH access to a remote host.
+
+The RHOST variable must be set to the hostname of the server (test management link) Which is taken care by the ltp script
+
+SSH server needs to be configured to allow root login and use Public Key Authentication (PermitRootLogin yes and PubkeyAuthentication yes in /etc/ssh/sshd_config).
```

Comment:
is this prerequisite or test already doing this, in that case this need to removed ?

Reply:
test already takes care of it

Reply:
All the things are already taken care in the script. This is just to inform user about configurations.

Reply:
in that case, i don't feel we need to have README we can have class doc string or reword README as it making like user need to have action